Sources/Core/AWSClientRuntime/Errors/UnknownAWSHTTPServiceError.swift

```diff
@@ -6,7 +6,8 @@
 import ClientRuntime
 
 /// AWS specific Service Error structure used when exact error could not be deduced from the `HttpResponse`
-public struct UnknownAWSHTTPServiceError: AWSServiceError, HTTPError, Error {
+/// Developers should catch unknown errors by the interface `AWSServiceError`, then use the `errorCode` to determine & handle each type of error.
+@_spi(Internal) public struct UnknownAWSHTTPServiceError: AWSServiceError, HTTPError, Error {
```

Comment:
Makes this struct part of SPI, with "Internal" tag.

Reply:
This needs a more specific name than "Internal".  Reason is that a single module may have multiple SPI's for different functions, so each function needs a specific name that allows the importer to only import the SPI they need or care about.

I'd suggest the type name `UnknownAWSHTTPServiceError` as your SPI identifier, since that's basically what you're concealing.

Reply:
Changed as suggested.